backend/src/app/rpc/commands/files.clj

```diff
@@ -474,30 +474,40 @@
   (update page :objects update-vals #(dissoc % :thumbnail)))
 
 (defn get-page
-  [{:keys [::db/conn] :as cfg} {:keys [profile-id file-id page-id object-id] :as params}]
+  [{:keys [::db/conn] :as cfg} {:keys [profile-id file-id page-id object-id share-id] :as params}]
 
   (when (and (uuid? object-id)
              (not (uuid? page-id)))
     (ex/raise :type :validation
               :code :params-validation
               :hint "page-id is required when object-id is provided"))
 
-  (let [team (teams/get-team conn
-                             :profile-id profile-id
-                             :file-id file-id)
+  (let [perms (get-permissions conn profile-id file-id share-id)
+
+        file  (get-file cfg file-id)
+
+        proj  (db/get* conn :project {:id (:project-id file)}
```

Comment:
I think this should be normal get instead of get* and without the `{::db/remove-deleted ...}`
The same for the team

Reply:
Thank you very much!